usb.cpp

```diff
@@ -40,6 +40,10 @@ namespace bsp
         constexpr inline auto usbCDCEchoOffCmdLength = usbCDCEchoOffCmd.length();
 #endif
 
+        void clearUSBSerialBuffer(){
+            memset(usbSerialBuffer,0,SERIAL_BUFFER_LEN);
```

Comment:
```suggestion
            memset(usbSerialBuffer, 0, sizeof usbSerialBuffer);
```

IMO it's a bit safer in case someone decides to arbitrarily hardcode the buffer size and not update the macro value.